debian/changelog

```diff
@@ -1,3 +1,9 @@
+apipe-metrics-daemon (1.1.35) lucid; urgency=low
+
+  * add metrics for info_prod_aligner disk group
+
+ -- Jim Eldred <jeldred@genome.wustl.edu> Wed, 3 Feb 2016 12:46:50 -0600
```

Comment:
The [Debian policy](https://www.debian.org/doc/debian-policy/ch-source.html) requires two spaces between the e-mail address and the date.  Furthermore, the day of the month should be two digits.